README.md

```diff
@@ -82,6 +82,8 @@ docker run --gpus all --shm-size 1g -p 8080:80 -v $volume:/data \
     ghcr.io/predibase/lorax:latest --model-id $model
 ```
 
+Please note that the LoRAX server in our pre-built Docker image is configured to listen on port 80 (instead of on the default port number, which is 3000).
```

Comment:
I like to keep quickstart guides very short and to the point to avoid too much cognitive load. Can we add this detail to the Docker section of the docs: https://predibase.github.io/lorax/getting_started/docker/

Reply:
@tgaddair This has been corrected, and here is how it looks in `mkdocs`.
![Screenshot 2024-03-08 at 11 10 42](https://github.com/predibase/lorax/assets/339166/d9a2dc59-4361-496a-90ea-a17f0207a4f8)

Thank you.